conf/test_full.config

```diff
@@ -41,7 +46,21 @@ process {
         if(System.getenv('GITHUB_ACTION') != null ) {
```

Comment:
Same as above 👆 
Everything under `process` needs to be moved to `conf/modules.config`. If it is in `conf/test_full.config` your tests will run correctly, but when you actually get to production, it will fail. Everything that you want implemented in production like the `longranger` arguments should be moved to `modules.config` and removed from here as it would be redundant.